UnattendResources/Logon.ps1

```diff
@@ -466,12 +470,19 @@ try {
     Release-IP
 
     if ($enableShutdownWithoutLogon) {
-       Set-ItemProperty "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System\" `
+        Set-ItemProperty "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System\" `
            -Name shutdownwithoutlogon -Value 1 -Type DWord
-       Set-ItemProperty "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Windows\" `
+        Set-ItemProperty "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Windows\" `
            -Name ShutdownWarningDialogTimeout -Value 1 -Type DWord
     }
 
+    # Determine if we are running Windows Client 6.2 or greater

```

Comment:
add this as a method please:

Disable-FirstLogonAnimation and run it only If disableFirstLogonAnimation is set, and then check for versions etc